### PR TITLE
ENH: RFC Template

### PR DESCRIPTION
**Pull request by thermokarst**
More or less the same as the scikit-bio-rfcs template.

### CODE REVIEW

**Comment by jakereps** [comment]
- [x] Add comments to YAML frontmatter with the QIIME 2 development team license header

File: 0000-template.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# Summary
+
+A brief (one paragraph) explanation of the proposed change.
+
+
+# Motivation
+
+Why are we doing this? What use cases does it support? What is the expected
+outcome?
+
```
Comment (ebolyen) [inline 0000-template.md]: Add a Glossary section.

File: 0000-template.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+# Detailed design
+
+This is the bulk of the RFC. Explain the design in enough detail for somebody
+familiar with QIIME 2, and for somebody familiar with the implementation to
+implement. This should get into specifics and corner-cases,
+and include examples of how the feature is used. Any new terminology should be
+defined here.
```
Comment (ebolyen) [inline 0000-template.md]: Move last sentence to Glossary

File: 0000-template.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- QIIME 2 Issue: (leave this empty)
+
+# Summary
+
+A brief (one paragraph) explanation of the proposed change.
+
+
```
Comment (ebolyen) [inline 0000-template.md:28]: Add Glossary

File: 0000-template.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
```
Comment (jairideout) [inline 0000-template.md]: Match scikit-bio's RFC frontmatter

**Comment by thermokarst** [comment]
Spell check and last glance look okay. I am done reviewing this.

File: 0000-template.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
+---
+# ----------------------------------------------------------------------------
+# This work is licensed under a Creative Commons Attribution 4.0 International
+# License. To view a copy of this license, visit:
+#
+# http://creativecommons.org/licenses/by/4.0/
```
Comment (jairideout) [inline 0000-template.md:6]: @gregcaporaso we were unsure of what license to use. We were previously using CC for this repo and skbio. Rust doesn't have a license but they are considering MIT or Apache licenses for their RFCs (both are permissive software licenses). It's unclear whether a permissive software license (BSD-3-clause) or CC is better for RFCs.

File: 0000-template.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+# ----------------------------------------------------------------------------
+Feature name: <unique id; lowercase; hyphenated>
+Start date: <YYYY-MM-DD>
+Pull request: <leave empty>
+Authors:
+  - "@<github-username>"
+  - "@<github-username>"
+  - ...
+Contributors:
+  - "@<github-username>"
```
Comment (jairideout) [inline 0000-template.md]: The guide will fill this in, can you update the template to indicate this section should not be filled in by the author?

File: 0000-template.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+Terms introduced in this RFC should be defined here.
+
+# Motivation
+
+Why are we doing this? What use cases does it support? What is the expected
+outcome?
+
+# Impact
+
+How will the changes in this RFC affect the following audiences: users, plugin
```
Comment (jairideout) [inline 0000-template.md]: This doesn't flow well with the rest of the doc and it's a sentence worded as a question. Can you update to:

_How will the changes in this RFC affect users, plugin developers, and interface/platform developers?_

**Comment by jairideout** [comment]
@thermokarst looks great, just a couple comments.

**Comment by ebolyen** [comment]
Looks good to me as well, pending @jairideout's comments.

**Comment by jakereps** [comment]
Looks good 👍

**Comment by thermokarst** [comment]
Thanks gang (or are we a posse?) - addressed comments.

**Comment by jairideout** [comment]
Looks good, this is ready for @gregcaporaso to review.

File: 0000-template.md (hunk 5 above)
Comment (gregcaporaso) [inline 0000-template.md:6]: I think this Creative Commons license make sense to use here. I think it gets us what we need, but I'm not an expert in the different licenses. I'd be ok with just going with this license.

File: 0000-template.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+# Impact
+
+How will the changes in this RFC affect users, plugin developers, and
+interface/platform developers?
+
+# Detailed design
+
+This is the bulk of the RFC. Explain the design in enough detail for somebody
+familiar with QIIME 2 to implement. You should expect to include concrete
+examples of how the feature is used.
```
Comment (gregcaporaso) [inline 0000-template.md]: We should include a note that prototyping ideas before writing this section is highly encouraged, and maybe a few suggestions on how to do that (e.g., in a new github repo, in a branch of a fork of the relevant QIIME 2 repo, etc). It's also worth noting here that QIIME 2 was built following a series of prototypes which were hugely beneficial to the design, and mention metoo and q2d2 (or even link to them, so it's clear how much work went into the QIIME 2 design).

**Comment by gregcaporaso** [comment]
Looks good to me too, only one comment that I think is important to address.

File: 0000-template.md (hunk 5 above)
Comment (jairideout) [inline 0000-template.md:6]: Sounds good

**Comment by thermokarst** [comment]
Recording for posterity: developed with @jakereps, @jairideout, @ebolyen, and @gregcaporaso

**Pull request merged**